src/test/java/org/apache/commons/collections4/bloomfilter/AbstractBitCountProducerTest.java

```diff
@@ -16,35 +16,29 @@
  */
 package org.apache.commons.collections4.bloomfilter;
 
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import org.apache.commons.collections4.bloomfilter.BitCountProducer.BitCountConsumer;
 import org.junit.jupiter.api.Test;
 
 public abstract class AbstractBitCountProducerTest extends AbstractIndexProducerTest {
-
+    /**
+     * A testing BitCountConsumer that always returns true.
+     */
+    private static final BitCountConsumer TRUE_CONSUMER = (i, j) -> true;
     /**
      * A testing BitCountConsumer that always returns false.
      */
-    public static BitCountConsumer FALSE_CONSUMER = new BitCountConsumer() {
-
-        @Override
-        public boolean test(int index, int count) {
-            return false;
-        }
-    };
+    private static final BitCountConsumer FALSE_CONSUMER = (i, j) -> false;
 
     /**
-     * A testing BitCountConsumer that always returns true.
+     * Creates an array of integer pairs comprising the index and the expected count for the index.
+     * @return an array of integer pairs comprising the index and the expected count for the index.
      */
-    public static BitCountConsumer TRUE_CONSUMER = new BitCountConsumer() {
-
-        @Override
-        public boolean test(int index, int count) {
-            return true;
-        }
-    };
+    protected abstract int[][] getExpectedBitCount();
```

Comment:
This method may not need to be abstract. In most cases where ordering and duplicates are not mandated then this implementation will be fine:
```Java
    protected int[][] getExpectedBitCount() {
        return Arrays.stream(getExpectedIndices()).mapToObj(x -> new int[] {x, 1}).toArray(int[][]::new);
    }
```